src/llamacrawl/services/query/synthesis.py

```
@@ -41,23 +42,23 @@ class AnswerSynthesizer:
 
     Attributes:
-        config: Configuration object with Claude settings
-        llm: ClaudeAgentLLM instance
-        model_name: Name of Claude model to use (e.g., "claude-sonnet-4-0")
+        config: Configuration object with synthesis settings
+        llm: QwenOllamaLLM instance
+        model_name: Name of model to use (from config.query.synthesis_model)
         max_input_tokens: Maximum number of tokens for context
     """
 
     def __init__(self, config: Config):
```

Ruff (0.14.0)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)